api/build.gradle

```diff
@@ -1,4 +1,5 @@
 description = 'OpenTelemetry API'
+ext.moduleName = "io.opentelemetry.api"
```

Comment:
we don't have "api" in the package names, does that matter?

Reply:
No, it doesn't matter. Module names do not need to relate to the package name, although they are a good starting point for choosing a good module name.